### DISCUSSION TITLE
Cant control device via one computer bus can from another

### DISCUSSION THREAD

**Discussion by Drumboyww** [Ylianst/MeshCentral#6612] [Issues]
I have a bit of a wierd one

A friend of me cant control pc A but can control pc B

I can control both pc's

But the perms are in a group
After a restart of pc A my friend can control it
But afer using it for sometime he cant control it anymore but i still can

The input checker is enabled on his computer 

Does someone know a fix?

Thankss

**Comment by si458** [comment]
what version of meshcentral? whats the remote computers OS? whats the browser you are using?
whats your config.json like? do you use reverse proxies?
a lot of question without answers...
